modules/nw-sriov-nic-mlx-secure-boot.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+[id="nw-sriov-nic-mlx-secure-boot_{context}"]
+= Mellanox Secure Boot
+
+Mellanox (MLX) secure boot supports the option to disable vendor plugins in the SR-IOV operator. This allows OCP customers to use the existing virtual functions in the mellanox cards firmware, and is currently the only way to use MLX NICs when secure boot is enabled. Secure boot is required by regulation and desired by customers.
+
+.Procedure
+
+. Configure the virtual functions (VFs)
+
+.. Run the following command when the system is without a secure boot when using the sriov-config daemon:
++
+[source,terminal]
+----
+mstconfig -d -0001:b1:00.0 set SRIOV_EN=1 NUM_OF_VFS=16
+----
+
+.. Reboot the system to enable the virtual functions.
+
+. Configure the sriov operator with the Mellanox plugin disabled
```

Comment:
```suggestion
. Configure the SR-IOV Network Operator with the Mellanox plugin disabled
```

Reply:
Thanks @cgoncalves for your feedback. I have applied your suggestion.